### ISSUE TITLE
RE: The GPS Example and Logging Data in .csv Format/Seth

### ISSUE THREAD

**Issue by silver2row** [beagleboard/cloud9-examples#49]
Hello,

I am having trouble w/ the source so far at https://github.com/beagleboard/cloud9-examples/blob/v2020.01/BeagleBone/Green/Grove/Software/Python/Grove_GPS.py. The Grove_GPS.py file states that it works on the github page but like I say, "trouble on my end."

So, I have changed some items in the source and came up empty. Are you aware of it not working thus far?

Seth

P.S. I have the GPS v1.2 from Seeed Studio and a BBGW w/ the updated image on it. I can get an older bunch of source to work in python2 for GPS but not this version for some reason.

**Comment by silver2row** [comment]
Hello Again,

To anyone reading this idea/issue, the Grove_GPS.py file works when I sign in on 192.168.7.2 on Win 10 in PuTTY at /var/lib/cloud9/BeagleBone/Green/Grove/Python/ but when it is run under another directory, it does not print output to the terminal. 

...

Is there a reason it only works in that file structure?

Seth

**Comment by jadonk** [comment]
Can you describe what you tried/expected a bit clearer?

**Comment by silver2row** [comment]
Yes Sir,

I plugged in the Grove connector GPS v1.2 to the UART on the BBGW. I made sure that /dev/ttyS2 was port used. 

I did not add the BB-UART2-00A0.dtbo to the uboot-overlays in /boot/uEnv.txt but I just transferred the file from that /var/lib/cloud9/ dir. to my /debian/home/ dir.

I ran it and received no output on the terminal. 

Seth

P.S. That is all I did. It works as is in the dir. of ../../cloud9/ but not in my dir. of choosing.
